appveyor.yml

```diff
@@ -23,17 +16,13 @@ build:
   parallel: true
   verbosity: minimal
 after_build:
-- ps: >-
-    Get-ChildItem ".\source\**\ScriptHookVDotNet*.csproj" | % { nuget pack $_.FullName }
-
-    Get-ChildItem ".\*.nupkg" | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
+- ps: ".\\tools\\package.ps1"
+- ps: ".\\tools\\upload_artifact.ps1"
 
 before_package:
 - cmd: >-
     del bin\ScriptHookVDotNet*.metagen
-
     type README.md | more /P > bin\README.txt
-
     type LICENSE.txt | more /P > bin\LICENSE.txt
```

Comment:
This doesn't work. The new lines are necessary or else AppVeyor will put these commands on a single line and fails to execute them:

https://ci.appveyor.com/project/crosire/scripthookvdotnet/builds/27846507
```
del bin\ScriptHookVDotNet*.metagen type README.md | more /P > bin\README.txt type LICENSE.txt | more /P > bin\LICENSE.txt
Cannot access file C:\projects\scripthookvdotnet\type
```